packages/wp-now/src/execute-php.ts

```diff
@@ -0,0 +1,36 @@
+import startWPNow from './wp-now';
+import { WPNowOptions } from './config';
+import { disableOutput } from './output';
+
+/**
+ *
+ * Execute a PHP cli given its parameters.
+ *
+ * @param phpArgs - Arguments to pass to the PHP cli. The first argument should be the string 'php'.
```

Comment:
Should we force this in some manner, or throw an exception if this isn't the case?

Reply:
Throwing an error https://github.com/WordPress/wordpress-playground/pull/451/commits/0a405abb8d010e400d09fa99bbe6077f2ed6de26